### PR TITLE
Change the default value of ui-brand to harvester

### PR DESCRIPTION
**Pull request by chrisho**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

**Problem:**
https://github.com/harvester/harvester/issues/4315

**Solution:**
Change the default value of ui-brand to "Harvester".

**Related Issue:**
https://github.com/harvester/harvester/issues/4315

**Test plan:**
Build and run the harvester image, watch the ui-brand setting, and its default value is "Harvester".

### CODE REVIEW

File: pkg/controller/master/rancher/embedded_rancher.go [hunk 1]
```diff
@@ -14,7 +14,8 @@ import (
 )
 
 var UpdateRancherUISettings = map[string]string{
-	"ui-pl": "Harvester",
+	"ui-pl":    "Harvester",
+	"ui-brand": "Harvester",
```
Comment (guangbochen) [inline pkg/controller/master/rancher/embedded_rancher.go]: It seems the UI only only recognize the lower case `harvester`, can we make it compatible for both @w13915984028?

**Comment by WuJun2016** [comment]
@chrisho   I tested the UI code and found that using capitalized `Harvester` for the UI was not very friendly, because it involved the case of the file and was used in many places, so I thought that this should continue to be the same as rancher and use lowercase `harvester`.    (for `ui-brand`).  cc @guangbochen

**Review by guangbochen — APPROVED** [review]
LGTM, thanks.

**Review by bk201 — APPROVED** [review]
LGTM!

**Comment by guangbochen** [comment]
@mergify backport v1.2

**Pull request merged**